README.md

```diff
@@ -17,14 +15,22 @@
 	 // to snapshot an entire ERC-721 collection using opensea
 	 "erc721addresses": [ "erc721contractAddress1", "erc721contractAddress2", ...],
 	 // if you want a specific range of token IDs from an ERC721
-	 "erc721TokenIDRanges": { "erc721contractAddress3": [0, 100], "erc721contractAddress4": [900, 10040] },
+	 // can duplicate contract addresses if multiple ranges within same contract
+	 "erc721TokenIDRanges": [
+		 ["erc721contractAddress3": [0, 100]],
+		 ["erc721contractAddress4": [900, 10040]]
+	 ],
```

Comment:
`erc721TokenIDRanges` takes an array of tuples instead of an object, so that we can duplicate contract address keys